scipy/special/_generate_pyx.py

```diff
@@ -1094,7 +1097,7 @@ def _generate_from_outargs_and_no_return(self):
 
             call = "{}({})".format(func_name, ", ".join(callvars))
             body.append(sp + call)
-            body.extend(map(lambda x: sp + x, casts))
+            body.extend([sp+x for x in casts])
```

Comment:
Are spaces around the operator required, `body.extend([sp + x for x in casts])`?

Reply:
added spaces